api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/ContentModelSetup.java

```diff
@@ -191,6 +213,144 @@ private void renamePortal(Resource portal, String namespace, Model model) {
         }
     }
 
+    /*
+     * Check if the firsttime files have changed since the firsttime startup,
+     * if so, then apply the changes but not overwrite the whole user model
+     */
+    private void applyFirstTimeChanges(ServletContext ctx) {
+        log.info("Start with check: Reload firsttime files on start-up if changed");
```

Comment:
Could you please change the logging message as follows:
```
log.info("Reload firsttime files on start-up if changed: '" + modelPath +"', URI: '" +modelUri+ "'");
```

Reply:
i changed it